bin/data-push.js

```diff
@@ -146,7 +146,16 @@ const prepareDatasetFromFile = async filePath => {
   // List of formats that are known as tabular
   const knownTabularFormats = ['csv', 'tsv', 'dsv']
   if (knownTabularFormats.includes(file.descriptor.format)) {
-    await file.addSchema()
+    try {
+      await file.addSchema()
+    } catch(err){
+      error("tabular file is invalid: " + file.path)
```

Comment:
What exactly can be invalid here? I guess `file.addSchema` throws error when tabular data is empty, right? What other situations?

Reply:
```
$ data push this_is_excel_file.csv --debug
> Error! Invalid opening quote at line 1
```

Reply:
I could imaging that `addSchema()` could fail for many reasons
- file is not readable
- this is actually not `csv`
- unknown separator ?

Reply:
@AcckiyGerman 
- `addSchema` would fail for non-tabular data
- it handles separators, if one of `, ; | ^ : * &`

Reply:
So do you want me to change the error message or leave it as is?